frame/transaction-payment/src/lib.rs

```diff
@@ -241,6 +241,10 @@ impl Default for Releases {
 	}
 }
 
+/// Default value for NextFeeMultiplier. This is used in genesis and is also used in
+/// NextFeeMultiplierOnEmpty() to provide a value when none exists in storage.
+const MULTIPLIER_DEFAULT_VALUE: Multiplier = Multiplier::from_u32(1);
```

Comment:
Please review, esp. that `from_u32(1)` is equivalent to `saturating_from_integer(1)`. The latter wasn't `const` and couldn't be used here.

My tests should verify this, but I'd appreciate another look at it.